noether_simulator/src/thickness_simulator_node.cpp

```diff
@@ -0,0 +1,247 @@
+#include "ros/ros.h"
```

Comment:
This should go in noether rather than noether_simulator. The reason is that we don't want these packages to depend on ROS so that we can use them as cmake packages and write ROS2 wrappers. I realize that noether needs some restructuring in general, but this should be fairly simple